compiler/test/test_end_to_end.re

```diff
@@ -920,7 +918,7 @@ let oom = [
 let gc = [
   tgc(
     "gc1",
-    256,
+    512,
```

Comment:
This previously was optimized into a single call of `f`, so now it makes more allocations than before. This change isn't an indication of the GC being broken, just the testing issue described in #504.